sumpy/fmm.py

```diff
@@ -415,6 +440,31 @@ def m2l_precomputed_exprs_view(self, m2l_precomputed_exprs, level):
         return (translation_class_start, exprs_level.reshape(
                             translation_class_stop - translation_class_start, -1))
 
+    @memoize_method
+    def pp_multipole_expansions_level_starts(self):
```

Comment:
```suggestion
    def m2l_preproc_mpole_expansions_level_starts(self):
``` 
(also below)

Reply:
Done in 5d32bfbe4853ea92e572e8c46f185e6084317dd5